gtk4/src/subclass/widget.rs

```diff
@@ -980,6 +980,24 @@ where
     }
 }
 
+impl<T> std::ops::Deref for TemplateChild<T>
+where
+    T: ObjectType + FromGlibPtrNone<*mut <T as ObjectType>::GlibType>,
+{
+    type Target = T;
+
+    /// # Safety
+    ///
+    /// Since the template child may not be properly bound,
```

Comment:
Can it be assumed that it will be impossible to call this from safe code when the template child isn't yet bound? Otherwise this is unsound.

Adding an assertion that it's non-null should be sufficient to avoid that, since it should be initialized to null by `default()`.

Reply:
Yes there should be an assertion here

Reply:
Added.